src/gui/application.cpp

```diff
@@ -306,6 +307,8 @@ void Application::slotAccountStateAdded(AccountState *accountState)
 
 void Application::slotCleanup()
 {
+    _isQuitting = true;
```

Comment:
Maybe we could simply do a specific disconnect([...]) here? See [Qt doc disconnect](https://doc.qt.io/qt-5/qobject.html#disconnect) and existing use for example in slotAccountStateRemoved().
As I glimpse through the code, there is generally no (intended) way back after entering slotCleanup(). Doing a disconnect here (before unloadAndDeleteAllFolders(), which happens itself directly before slotShutdown()) would simply prevent the trigger for the gui change. This would also have the benefit that can prevent any GUI reflections that are not wanted on shutdown, not only the wizard (e.g. icon change / disconnect)

Reply:
OK, I added a disconnect call before `AccountManager::instance()->shutdown()` in the destructor and removed the `_isQuitting` flag.

Reply:
Awesome, thanks!